libs/community/langchain_community/vectorstores/meilisearch.py

```diff
@@ -143,7 +131,7 @@ def add_texts(
             docs.append(
                 {
                     "id": id,
-                    "_vectors": {f"{embedder_name}": embedding},
+                    "_vectors": {f"{embedder}": embedding},
```

Comment:
Style: I think we can do without the interpolation here, given as `embedder` is already a `str`

```suggestion
                    "_vectors": {embedder: embedding},
```